docs/source/tutorials/otx_cli/how_to_train/detection.rst

```diff
@@ -303,16 +277,28 @@ Export
   2022-12-29 01:39:11,990 | INFO : Exporting completed
 
 
-#TODO show how to run evaluation of exported model?
+3. You can check the accuracy of exported model as simple as accuracy of the ``.pth`` model, using ``otx eval`` with the path of IR model.
+
+.. code-block::
+
+  (detection) ...$ otx eval otx/algorithms/detection/configs/detection/mobilenetv2_atss/template.yaml
+                            --test-ann-files BBCD/valid/_annotations.coco.json 
+                            --test-data-roots  BBCD/valid 
+                            --load-weights outputs/openvino/openvino.xml
+                            --save-performance outputs/openvino/performance.json
+  
+  ...
+
+
 
 *************
 Optimization
 *************
 
 1. ``otx optimize`` optimizes a model using NNCF or POT depending on the model format.
 
-- NNCF optimization used for trained snapshots in a framework-specific format such as checkpoint (pth) file from Pytorch. It starts training based on the weights from previous step in fewer precision.
-- POT optimization used for models exported in the OpenVINO IR format. It decreases the precision of the exported model and performs the post-training optimization.
+- NNCF optimization is used for trained snapshots in a framework-specific format such as checkpoint (pth) file from Pytorch. It starts training-aware quantization based on the obtained weights from the training stage.
```

Comment:
Sorry for misleading, according to the NNCF docs, the right naming is: Accuracy-Aware-training

```suggestion
- NNCF optimization is used for trained snapshots in a framework-specific format such as checkpoint (.pth) file from Pytorch. It starts Accuracy-Aware quantization based on the obtained weights from the training stage.
```